src/components/DateInput/index.js

```diff
@@ -6,43 +6,43 @@ export class DateInput extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            start: "2002-05-06",
-            end: Moment().format('YYYY-MM-DD'),
+            startDate: "2002-05-06",
+            endDate: Moment().format('YYYY-MM-DD'),
         }
     }
 
-    startChange = (e) => {
+    startDateChange = (e) => {
```

Comment:
A preferred convention for these handlers is to set a prefix. You will see some developers use `on`, others will user `handle`. Both are acceptable. So for these functions, consider changing them to something like:

```
onChangeDate = (e, key) => this.setState({ [key]: e.target.value });
```

then in the implementation, you can use the same function, but pass it a key:
```
onChange={(e) => this.onChangeDate(e, 'startDate')}
```